packages/cursorless-engine/src/actions/SimpleIdeCommandActions.ts

```diff
@@ -103,6 +103,11 @@ export class RevealDefinition extends SimpleIdeCommandAction {
   ensureSingleTarget = true;
   restoreSelection = false;
 }
+export class RevealDefinitionAside extends SimpleIdeCommandAction {
+  command: CommandId = "revealDefinitionAside";
+  ensureSingleTarget = true;
+  restoreSelection = false;
```

Comment:
I'd remove this so that it restores the selection
```suggestion
```

Reply:
Changed.